pkg/model/manifest.go

```diff
@@ -13,21 +13,24 @@
 
 package model
 
-import "github.com/okteto/okteto/pkg/log"
+import (
+	"github.com/okteto/okteto/pkg/log"
+)
 
 //Manifest represents an okteto manifest
 type Manifest struct {
-	Name    string       `json:"name,omitempty" yaml:"name,omitempty"`
-	Icon    string       `json:"icon,omitempty" yaml:"icon,omitempty"`
-	Deploy  *DeployInfo  `json:"deploy,omitempty" yaml:"deploy,omitempty"`
-	Dev     ManifestDevs `json:"devs,omitempty" yaml:"devs,omitempty"`
-	Destroy []string     `json:"destroy,omitempty" yaml:"destroy,omitempty"`
-
-	Type     string `yaml:"-"`
-	Filename string `yaml:"-"`
+	Name     string        `json:"name,omitempty" yaml:"name,omitempty"`
+	Icon     string        `json:"icon,omitempty" yaml:"icon,omitempty"`
+	Deploy   *DeployInfo   `json:"deploy,omitempty" yaml:"deploy,omitempty"`
+	Dev      ManifestDevs  `json:"devs,omitempty" yaml:"devs,omitempty"`
+	Destroy  []string      `json:"destroy,omitempty" yaml:"destroy,omitempty"`
+	Build    ManifestBuild `json:"build,omitempty" yaml:"build,omitempty"`
```

Comment:
is it possible to reuse `BuildInfo`?
https://github.com/okteto/okteto/blob/fd03d2c861691e15320dd99f63c97af18674cda7/pkg/model/dev.go#L116

Reply:
Build will still need to be a map for the different images defined, you mean changing the BuildParams struct for each image to BuildInfo?

Reply:
Right